bagpipes/fitting/prior.py

```
@@ -83,6 +83,12 @@ def uniform(self, value, limits, hyper_params):
         value = limits[0] + (limits[1] - limits[0])*value
         return value
 
+    def exponential(self, value, limits, hyper_params):
```

Ruff (0.14.5)

Unused method argument: `limits`

(ARG002)